Answers.md

```diff
@@ -1,3 +1,18 @@
-### Your responses to the short answer questions should be laid out here using markdown.
 
-For help with markdown syntax, [go here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+# Assessment Answers
+ ### 1. Describe the differences between a Docker container and a virtual machine. What makes a container more aptly-suited for portability?
+    Docker containers use a lot less resources and generally run faster than VMs. Dockers also don't have system overhead to worry about.
```

Comment:
More specifically, Docker doesn't contain an entire copy of an operating system, making it a lot more lightweight.